tools/restatectl/src/commands/node/list_nodes.rs

```diff
@@ -125,6 +186,28 @@ where
         .unwrap_or(vec![Cell::new("N/A"), Cell::new("Unknown")])
 }
 
+fn render_ident_extras(ident_response: &IdentResponse) -> Vec<Cell> {
+    vec![
+        duration_to_human_rough(
+            TimeDelta::seconds(ident_response.age_s as i64),
+            Tense::Present,
+        ),
+        // We reuse the prost-generated Debug formatting but cut out the "Unknown"s
+        format!("{:?}", ident_response.status()).replace("Unknown", "-"),
+        format!("{:?}", ident_response.admin_status()).replace("Unknown", "-"),
+        format!("{:?}", ident_response.worker_status()).replace("Unknown", "-"),
+        format!("{:?}", ident_response.log_server_status()).replace("Unknown", "-"),
+        format!("{:?}", ident_response.metadata_server_status()).replace("Unknown", "-"),
+        format!("v{}", ident_response.nodes_config_version),
+        format!("v{}", ident_response.logs_version),
+        format!("v{}", ident_response.schema_version),
+        format!("v{}", ident_response.partition_table_version),
+    ]
+    .iter()
```

Comment:
This might save us few allocations.

```suggestion
    .into_iter()
```